api/v2/utilities/logs_validator.py

```diff
@@ -0,0 +1,36 @@
+from flask import request, jsonify, make_response
+from functools import wraps
+from cerberus import Validator
+
+
+def validate_log_data(func):
+    '''
+    This is a decorator to validate log data from the request
+    '''
+    @wraps(func)
+    def wrapper():
+        schema = {
+            "calendar_id" : {'type': 'integer'},
+            "subscriber_name" : {'type': 'string', 'min': 3},
+            "subscription_method" : {'type': 'string', 'min': 3},
+            "payload" : {'type': 'string', 'min': 3},
+        }
+        if func.__name__ == 'post':
+            data = request.get_json()
+            try:
```

Comment:
This is some really pretty work but I was able to create logs with an empty string, I think adding some validation to curb such bugs would be really nice and security wise

Reply:
Good eye, let me add that validation